pytorch_lightning/callbacks/rich_model_summary.py

```diff
@@ -88,6 +88,8 @@ def summarize(
                 table.add_column(column_name, justify="right", style="white")
 
         rows = list(zip(*(arr[1] for arr in summary_data)))
+        # convert all elements to str for Rich
+        rows = list(list(str(x) for x in row) for row in rows)
```

Comment:
Should we convert it to str before passing it to the `summarize` method?

Reply:
That's a good idea, I just looked further down the trace and it seems we don't really enforce that the input/output sizes are strings. I'll modify this PR to reflect this instead!